README.md

```diff
@@ -194,7 +195,7 @@ When triggering ad hoc runs in your dev/sandbox environment, or when running the
 
 ## Testing
 
-Unit tests and end-to-end (E2E) pipeline tests are performed using [pytest](https://docs.pytest.org). 
+Unit tests and end-to-end (E2E) pipeline tests are performed using [pytest](https://docs.pytest.org).
```

Comment:
remove reference to E2E tests - `Unit tests are performed using [pytest](https://docs.pytest.org).`